docs/content/tutorial/step_07.ngdoc

```diff
@@ -1,380 +1,311 @@
 @ngdoc tutorial
-@name 7 - Routing & Multiple Views
+@name 7 - XHR & Dependency Injection
 @step 7
 @description
 
 <ul doc-tutorial-nav="7"></ul>
 
 
-In this step, you will learn how to create a layout template and how to build an app that has
-multiple views by adding routing, using an Angular module called 'ngRoute'.
+Enough of building an app with three phones in a hard-coded dataset! Let's fetch a larger dataset
+from our server using one of Angular's built-in {@link guide/services services} called
+{@link ng.$http $http}. We will use Angular's {@link guide/di dependency injection (DI)} to provide
```

Comment:
`{@link ng.$http}` will format the `$http` as an inline code block. As you have it now the $http will look like normal text. What were you aiming for?